docs/it/ciclo-del-feedback.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Ciclo del Feedback
+nav_order: 999
+---
+
+# Il Ciclo del Feedback
+
+## La flessibilità del software
+
+Intuitivamente parlando, sembra ovvio che noi sviluppatori, con una connessione internet e un portatile alla mano, possiamo essere enormemente più produttivi di decine di operai da catena di montaggio del secolo scorso. Ma perché è davvero così?
+
+Sicuramente un fattore è la possibilità di lavorare con strumenti frutto di decenni di evoluzione tecnologica esponenziale, ma la vera ragione, il vero discriminante, sta nel prefisso "soft" nella parola "software": il prodotto del nostro lavoro è spesso intangibile e facilmente manipolabile, tanto che se commettiamo un errore, non dobbiamo prendere il semilavorato e buttarlo via, sprecando materie prime, ma nella peggiore delle ipotesi abbiamo solo sprecato un po' di tempo.
```

Comment:
manipolabile, tanto che se commettiamo un errore, -> manipolabile tanto che, se commettiamo un errore,

sprecando materie prime, ma nella peggiore delle ipotesi abbiamo solo sprecato un po' di tempo  -> sprecando materie prime ma, nella peggiore delle ipotesi, abbiamo solo sprecato un po' di tempo

Reply:
Fatto in fix: incorpora suggerimenti di review Alessandro Lai Moments ago